sdk/go/tappd/client_test.go

```diff
@@ -124,7 +124,7 @@ func TestTdxQuoteRawHash(t *testing.T) {
 
 	// Test too large raw hash
 	largeData := make([]byte, 65)
-	_, err = client.TdxQuote(context.Background(), largeData, tappd.RAW)
+	_, err = client.TdxQuote(context.Background(), largeData)
```

Comment:
The change is unnecessary because the length check applies only to `RAW`.

Reply:
Makes total sense, ended up skipping this - tests failed anyway. Fixed in [9eaf1dc](https://github.com/Dstack-TEE/dstack/pull/47/commits/9eaf1dc20619561541f580b56fedfb574f8d345d)